common/tools/turborepo-remote-cache/Dockerfile

```diff
@@ -1,11 +1,11 @@
 # Use an official node 22-slim image
-FROM node:22-slim
```

Comment:
Update the comment above about which image we are taking

Reply:
updated to just have 22